### PR TITLE
For GH-497: conference tweaks for fallout from schedule/typography

### PR DESCRIPTION
**Pull request by mewtaylor**
Thanks @carljbowman and @jwzimmer ! Here are some updates for typography/mobile fixes

### CODE REVIEW

File: src/components/footer/conference/footer.scss [hunk 1]
```diff
@@ -50,7 +50,7 @@
                 }
 
                 li {
-                    margin-bottom: .5rem;
+                    margin: .0;
```
Comment (rschamp) [inline src/components/footer/conference/footer.scss]: Lint

Comment (mewtaylor) [inline src/components/footer/conference/footer.scss]: thanks!

File: src/views/conference/index/index.jsx [hunk 2]
```diff
@@ -21,10 +21,17 @@ var ConferenceSplash = React.createClass({
                     <h3>
                         Scratch Conference 2016 | Cambridge, MA, USA
                     </h3>
+                    <h3>
+                        August 4 - 6
```
Comment (carljbowman) [inline src/views/conference/index/index.jsx]: Let's move this up and take out 'Scratch Conference' so it reads:

'August 4–6, 2016 | Cambridge, MA, USA'

File: src/views/conference/index/index.jsx [hunk 3]
```diff
@@ -21,10 +21,17 @@ var ConferenceSplash = React.createClass({
...
+                    </h3>
                     <p>
-                        <a href="//scratchconference2016.eventbrite.com">
+                        <b>
+                            We're sold out! <a href="//scratchconference2016.eventbrite.com">Join the Waitlist</a>
```
Comment (carljbowman) [inline src/views/conference/index/index.jsx]: Let's move this line below the button with some spacing.

**Comment by carljbowman** [comment]
@mewtaylor - sorry for the headache. thanks for all these changes!!! 

Looks good to me!

**Pull request merged**